includes/discovery/sensors/ipmi.inc.php

```diff
@@ -28,7 +28,7 @@
             $results = explode(PHP_EOL, external_exec(array_merge($cmd, ['-I', $ipmi_type, 'sensor'])));
 
             $results = array_values(array_filter($results, function ($line) {
-                return ! Str::contains($line, 'discrete');
+                return ! Str::contains($line, 'discrete') && trim($line) !== '';
```

Comment:
This patch causes problems with my voltage thresholds, as described in issue #18274.